index.js

```diff
@@ -38,8 +38,51 @@ alfy.alfred = {
 
 alfy.input = process.argv[2];
 
-alfy.output = arr => {
-	console.log(JSON.stringify({items: arr}, null, '\t'));
+const isDefined = x => x !== null && x !== undefined;
+
+const wrapArg = item => {
+	const alfredworkflow = {arg: item.arg, variables: item.variables};
+	const arg = JSON.stringify({alfredworkflow});
+	const newItem = Object.assign({}, item, {arg});
```

Comment:
I see your reasoning. But I copied the item because I wanted to avoid this kind of scenario:

```js
const unicorn = {
  title: 'unicorn',
  arg: '🦄',
  variables: {
    selectedMagicalBeing: true
  }
};

alfy.output([unicorn]);

const unicornEmoji = unicorn.arg;
doSomething(unicornEmoji);
// weird behavior. unicornEmoji is now a stringified JSON object
```

Obviously, you can avoid this sort of problem by calling `alfy.output` last. And I'm sure that's how most people use it. I just don't want Alfy users to have to remember to call `alfy.output` last.